_includes/proposals.html

```diff
@@ -2,10 +2,10 @@
   <a id="proposals" tabindex="-1"></a>
   <h2 class="section-headline text-end">State of Proposals</h2>
   <div class="featurelist">
-    <h3>Stage 3</h3>
+    <h3>Upcoming additions to the Specification</h3>
     <i class="featurelist__desc">
-      Proposals that have been reviewed, and are being considered as
-      candidates for the spec are considered to be stage 3.
+      This section features proposals that are close to completion. We call these "stage 3"
```

Comment:
“We call these” makes it sound like we don’t have a non-vague process that defines it; what’s the reason to make this more vague?

Reply:
I should clarify -- the commit message is not clear.

It was mentioned by someone on the committee that "Stage 3" is meaningless to folks who are not in the committee, and just browsing the website. The goal of this blurb is to contextualize this set of proposals in some way, and inform people that there is such a thing called "stage 3" but what that means is that they are close to completion.

Reply:
Can we do that by linking to the process document, or to a different page on this website that contextualizes what the (concrete, not vague) stages mean?

I totally ack that “stage X” is meaningless to many, but that’s not the same as it being a casual nickname we use to categorize proposals.

Reply:
is the issue that you are raising that stage 3 is in quotations? would the original blurb work better for you?

The stages are already contextualized above the fold, I can add a link here as well.

Reply:
It’s that the phrase “we call” connoted to me that it is an arbitrary and casual nickname, as opposed to a tightly defined stage in our process.

Reply:
Maybe: `In [our process], these are referered to as "stage 3" proposals.`

Reply:
```suggestion
      This section features proposals that are in Stage 3 of [our process](#link-to-process-here), which means they are close to completion and likely ready to be shipped in engines.
```
?